runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc

```diff
@@ -365,7 +365,9 @@ OperationFactory::OperationFactory()
     OperandIndexSequence inputs{init_param.inputs[0], init_param.inputs[1]};
     OperandIndexSequence outputs{init_param.outputs[0]};
 
-    return new operation::Reshape{inputs, outputs};
+    operation::Reshape::Param param{};
+
+    return new operation::Reshape{inputs, outputs, param};
```

Comment:
@periannath You don't initialize `param.new_shape`. To complete `nnapi` frontend, you have to modify `nnapi_delegate.cpp` also.

Reply:
@glistening I've checked NNAPI spec and `new_shape` option is not specified in NNAPI. I just added params at there since constructor of `Reshape` needs argument param. If `new_shape` option is not specified, reshape will use second input as shape of output tensor.